plugins/modules/docker_container.py

```diff
@@ -1750,10 +1759,9 @@ def _parse_publish_ports(self):
             if len(self.published_ports) > 1:
                 self.client.module.deprecate(
                     'Specifying "all" in published_ports together with port mappings is not properly '
-                    'supported by the module. The port mappings are currently ignored. Please specify '
-                    'only port mappings, or the value "all". The behavior for mixed usage will either '
-                    'be forbidden in version 2.0.0, or properly handled. In any case, the way you '
-                    'currently use the module will change in a breaking way',
+                    'supported by the module. The port mappings are currently ignored. Set publish_all_ports '
+                    'to "true" to randomly assign port mappings for those not specified by published_ports. '
+                    'The use of "all" in published_ports will be removed in version 2.0.0.',
```

Comment:
```suggestion
                    'The use of "all" in published_ports next to other values will be removed in version 2.0.0.',
```
`all` will still be allowed in 2.0.0. What we will do in 2.0.0 is deprecate `all` (with removal in 3.0.0). But that shouldn't be mentioned here.